CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### TODO add version
```

Comment:
can this be reverted? The changelog is generated automatically from commits on release.

Reply:
Reverted